tests/release/release.go

```diff
@@ -61,12 +62,12 @@ var _ = framework.ReleaseSuiteDescribe("test-demo", func() {
 
 	var _ = Describe("Creation of the 'Happy path' resources", func() {
 		It("Create an ApplicationSnapshot.", func() {
-			_, err := framework.ReleaseController.CreateApplicationSnapshot(snapshotName, devNamespace, applicationName, snapshotImages)
+			_, err := framework.ReleaseController.CreateApplicationSnapshot(snapshotName, devNamespace, applicationName, snapshotComponents)
 			Expect(err).NotTo(HaveOccurred())
 		})
 
 		It("Create Release Strategy", func() {
-			_, err := framework.ReleaseController.CreateReleaseStrategy(releaseStrategyName, managedNamespace, releasePipelineName, releasePipelineBundle)
+			_, err := framework.ReleaseController.CreateReleaseStrategy(releaseStrategyName, managedNamespace, releasePipelineName, releasePipelineBundle, releaseStrategyPolicy)
```

Comment:
I think this change is unrelated to this PR and should probably go into a different PR. In any case, this is probably difficult to do it in two different PRs as both changes are needed to pass the tests.